backend/src/models/Schema.ts

```diff
@@ -5,7 +5,7 @@ import { SchemaType } from '../types/types.js'
 export interface Schema {
   name: string
   reference: string
-  schema: unknown
+  schema: any
```

Comment:
`unknown` is better than `any` as it forces us to assert that properties of an unknown type exist before accessing them. `any` simply turns off TypeScript.